src/components/nodes/pages/sup-book/digital-project-page/digital-project-page.tsx

```diff
@@ -42,214 +42,225 @@ const DigitalProjectPage = async ({node, ...props}: Props) => {
   const bookSubjectLinkParams = node.supBookSubjects && createLinkParams(node.supBookSubjects[0])
 
   return (
-    <article className="centered" {...props}>
-      <>
-        <div className="mb-20 flex flex-col md:flex-row md:gap-24 md:pt-32">
-          <div className="relative left-1/2 flex w-screen -translate-x-1/2 flex-row justify-center gap-24 bg-fog-light px-20 pt-32 md:hidden">
-            <div className="mb-16 hidden w-8/12 flex-col sm:flex md:hidden">
-              <H1 className="type-3 mb-0">{node.title}</H1>
-
-              {node.supBookSubjects && (
-                <a
-                  href={`/search?${bookSubjectLinkParams}`}
-                  className="rs-mb-2 -type-2 order-first font-normal text-stone-dark decoration-fog-dark underline-offset-[5px] hocus:text-archway-dark hocus:decoration-archway-dark hocus:decoration-2"
-                >
-                  {node.supBookSubjects[0].parent?.name || node.supBookSubjects[0].name}
-                </a>
-              )}
+    <div className="centered flex flex-col">
+      <article className="centered order-last" {...props}>
+        <>
+          <div className="mb-20 flex flex-col md:flex-row md:gap-24 md:pt-32">
+            <div className="relative left-1/2 flex w-screen -translate-x-1/2 flex-row justify-center gap-24 bg-fog-light px-20 pt-32 md:hidden">
+              <div className="mb-16 hidden w-8/12 flex-col sm:flex md:hidden">
+                <H1 className="type-3 mb-0">{node.title}</H1>
 
-              {node.supBookSubtitle && <div className="type-2 mt-5 font-medium">{node.supBookSubtitle}</div>}
+                {node.supBookSubjects && (
+                  <a
+                    href={`/search?${bookSubjectLinkParams}`}
+                    className="rs-mb-2 order-first text-18 font-normal text-stone-dark decoration-fog-dark underline-offset-[5px] hocus:text-archway-dark hocus:decoration-archway-dark hocus:decoration-2"
+                  >
+                    {node.supBookSubjects[0].parent?.name || node.supBookSubjects[0].name}
+                  </a>
+                )}
 
-              {node.supBookAuthorsFull && (
-                <div className="type-1 mt-5 text-press-sand-dark">{node.supBookAuthorsFull}</div>
-              )}
-            </div>
-            <div className="order-first w-5/12">
-              <BookPageImage node={node} />
+                {node.supBookSubtitle && <div className="type-0 mt-5 font-medium">{node.supBookSubtitle}</div>}
+
+                {node.supBookAuthorsFull && (
+                  <div className="type-1 mt-5 text-press-sand-dark">{node.supBookAuthorsFull}</div>
+                )}
+              </div>
+              <div className="order-first w-5/12">
+                <BookPageImage node={node} />
```

Comment:
I believe they wanted to use a landscape image for the digital projects pages. Is that still the case? they emailed about it for the search results as well.

Reply:
Yeah, i actually just noticed that and committed a fix 12 minutes ago :smile: